bibliopixel/led.py

```diff
@@ -506,7 +506,7 @@ def push_to_driver(self, frameTime=None):
             def color(i):
                 return self._colors[(h + width * i) * 3]
             buf = [color(i) for i in range(self.height)]
-            self.drivers[0].update_colors()
+            self.drivers[0].set_colors(buf, 0)
```

Comment:
<shrug> buf was set in line 508 and never used. Honestly that wasn't tested but it didn't in fact work... so I was guessing. Just doing cleanup of somewhat logical things, but then a LOT of testing is needed.